mso/datasource_mso_schema_site_bd.go

```diff
@@ -66,21 +70,23 @@ func dataSourceMSOSchemaSiteBdRead(d *schema.ResourceData, m interface{}) error
 		return fmt.Errorf("No Sites found")
 	}
 	stateSite := d.Get("site_id").(string)
+	stateTemplate := d.Get("template_name").(string)
```

Comment:
if template name is used the schema attribute should be changed to required attribute

Reply:
ForceNew: true, not required in data source

Reply:
ok